features/issue/components/EditIssueForm.tsx

```diff
@@ -196,6 +218,25 @@ export const EditIssueForm = () => {
 								</FormItem>
 							)}
 						/>
+						{!isUsersPending ? (
+							<MultiSelect
+								label='Friends'
+								name='users'
+								isMulti={true}
+								control={control}
+								isSearchable={true}
+								defaultValue={issueUsers.map(user => ({
+									label: user.name,
+									value: user.id,
+								}))}
+								options={friendsSelectOptions}
+							/>
+						) : (
+							<div>
+								<p>Friends are being fetched...</p>
```

Comment:
Jak ci się chce jebać w ładniejszą obsługe loading state'ów to tutaj fajny komponent z shadcn'a
https://ui.shadcn.com/docs/components/skeleton

Reply:
Bo <p>Friends are being fetched</p> słabe jest, ale wiadomo nie chce się